README.md

```
@@ -31,6 +42,44 @@ It could be useful later for the organization (community, managers, etc.) to acc
 https://github.com/gnolang/gno
 
+---
+
+### Getting started
+1. Clone the repository
+`git clone https://github.com/samouraiworld/gnolove.git`
+
+2. Set up environment variables
+  `cp .env.example .env`
+  Required environment variables:
+  - `GITHUB_API_TOKEN`: Your GitHub API token. Create one at https://github.com/settings/tokens
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)